### DISCUSSION TITLE
Did output_format='json' replace json_output=True

### DISCUSSION THREAD

**Discussion by BreezySMX** [adbar/trafilatura#205] [Q&A] [answered]
Python version 3.10.1
trafilatura version 1.2.1

I'm using the tutorial from: https://sempioneer.com/python-for-seo/how-to-extract-text-from-multiple-webpages-in-python/

Running the code presents the following error:

`TypeError: extract() got an unexpected keyword argument 'json_output'`

from this line of code:
```
   a = trafilatura.extract(downloaded_url, json_output=True, with_metadata=True, include_comments = False,
                            date_extraction_params={'extensive_search': True, 'original_date': True})
```

I appear not to be the only one experiencing this issue:

- https://solveforum.com/forums/threads/solved-typeerror-xxxxx-got-an-unexpected-keyword-argument-xxxxxx.282347/
- https://exerror.com/typeerror-extract-got-an-unexpected-keyword-argument-json_output/

I went to the documents to see if I could figure out what the issue is.  [Documentation](https://trafilatura.readthedocs.io/en/latest/corefunctions.html?highlight=extract()#trafilatura.extract)

From here I see trafilatura.extract does not have "json_output".  However, I do see "output_format".  Are these interchangeable?

The following seems to work:

```
    a = trafilatura.extract(downloaded_url, output_format='json', with_metadata=True, include_comments = False,
                            date_extraction_params={'extensive_search': True, 'original_date': True})
```

However I want to be sure. I don't see anything in the Github changelog referencing this.

Thank you

**Comment by adbar** [comment, marked as answer]
Hi @BreezySMX, thanks for your feedback!

These arguments are now indeed deprecated in favor of `output_format` but the information might not be so easy to find. I added a corresponding error in 24aa207742c9d2a9317f6ce15842ef32bd891e56.
